wikimedia/wikipedia-ios
# Native Editor - Add close confirmation

**Phabricator:** 
https://phabricator.wikimedia.org/T348092

### Notes
This is a quick copy of our destructive alert from the legacy editor.

### Test Steps
1. Open editor on Staging
2. Close, confirm you see alert confirmation and it works as expected.
